projects/desktop/src/components/atoms/singleForm/input/Input.tsx

```diff
@@ -85,6 +85,9 @@ export const Input = (props: IInputProps): JSX.Element => {
                         <InputField
                             {...rest}
                             value={value}
+                            InputProps={{
```

Comment:
What is it for?

Reply:
This is the most important part, we can customize the `Input` (first field component in mui) with some props.

Here we use `inputComponent` to pass our masked input.

This allows us to not recreate a complete field just(nob) for that.

Reply:
Nooo justnobbbbbb